### PR TITLE
Add validation endpoint process model

### PR DESCRIPTION
**Pull request by lucianoprea**
refs [#1829](https://github.com/Activiti/Activiti/issues/1829)

### CODE REVIEW

File: activiti-cloud-services-org/activiti-cloud-services-org-core/pom.xml [hunk 1]
```diff
@@ -22,5 +21,10 @@
       <groupId>org.springframework.data</groupId>
       <artifactId>spring-data-rest-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.activiti</groupId>
+      <artifactId>activiti-process-validation</artifactId>
+      <version>${version}</version>
```
Comment (mteodori) [inline activiti-cloud-services-org/activiti-cloud-services-org-core/pom.xml]: version should be set by parent and removed here

File: activiti-cloud-services-org/activiti-cloud-services-org-rest/pom.xml [hunk 2]
```diff
@@ -129,5 +128,10 @@
       <artifactId>spring-cloud-starter-contract-verifier</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.activiti.cloud</groupId>
+      <artifactId>activiti-cloud-services-org-core</artifactId>
+      <version>${version}</version>
```
Comment (mteodori) [inline activiti-cloud-services-org/activiti-cloud-services-org-rest/pom.xml]: version is set by parent and can be removed here

File: activiti-cloud-services-org/activiti-cloud-services-org-core/pom.xml (hunk 1 above)
Comment (lucianoprea) [reply activiti-cloud-services-org/activiti-cloud-services-org-core/pom.xml]: done

File: activiti-cloud-services-org/activiti-cloud-services-org-rest/pom.xml (hunk 2 above)
Comment (lucianoprea) [reply activiti-cloud-services-org/activiti-cloud-services-org-rest/pom.xml]: done

**Comment by lucianoprea** [comment]
resolved the requested changes and also rebased this to resolve the merge conflicts

**Pull request merged**